pkg/networking/handler.go

```diff
@@ -0,0 +1,13 @@
+package networking
+
+// Handler is an interface for handling new messages, handshakes and session close events.
+type Handler interface {
+	// OnReceive fired on new message received.
+	OnReceive(*Session, []byte)
```

Comment:
Yes, because there is no such interface. Do we need it? Actually `Session` combines a few interfaces, such as `io.Writer`, `io.Closer`, `Stringer` and non-public `addressable`. Hard to imagine which functions of `Session` will be required in handlers. Obviously it's `Writer` to send replies. And `Addressable` for checks and logging. What else? Doubt about `Closer`. Should we introduce a `Session` interface?

Reply:
Fair enough, resolving